vktrace/vktrace_layer/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ cmake_minimum_required(VERSION 2.8)
 if(CMAKE_SYSTEM_NAME STREQUAL "Linux")
    if (BUILD_WSI_XCB_SUPPORT)
        set(ENV{VULKAN_WSI} "Xcb")
-       pkg_search_module(XCB-KEYSYMS REQUIRED xcb-keysyms)
+       find_library(XCB-KEYSYMS REQUIRED xcb-keysyms)
```

Comment:
find_package  is probably preferred here.   find_package will set vars that help CMake locate headers and libraries, whereas find_library will set vars that help CMake locate just the libraries.

I'd suggest trying find_package and use it if it works.